less/themes.less

```diff
@@ -124,58 +124,60 @@
   --textarea-border__focus: var(--input_field-border__focus);
 
   // Tab Headers
-  --tabHeader-bg: transparent; //hsl(230, 1%, 92%);
-  --tabHeader-box-shadow: hsl(230deg 1% 86%);
-  --tabBtn-color: hsl(230, 1%, 14%);
-  --tabBtn-border: hsl(230, 1%, 86%);
-  --tabBtn-bg_active: hsl(230, 1%, 96%);
+  --tab-header-bg: transparent; //hsl(230, 1%, 92%);
+  --tab-header-box-shadow: hsl(230deg 1% 86%);
+  --tab-btn-color: hsl(230, 1%, 14%);
+  --tab-btn-border: hsl(230, 1%, 86%);
+  --tab-btn-bg_active: hsl(230, 1%, 96%);
 
-  --tabWrapper-border: hsl(230deg, 1%, 66%);
-  --tabContainer-bg: hsl(230, 1%, 96%);
+  --tab-wrapper-border: hsl(230deg, 1%, 66%);
+  --tab-container-bg: hsl(230, 1%, 96%);
 
   // Page Header
-  --pageHeader-bg: hsla(0, 100%, 100%, 0.9);
-  --pageHeader-border: hsl(230, 1%, 86%);
-  --pageHeader_link-color: hsl(0, 0%, 40%);
-  --pageHeader_box-shadow-color: rgba(0, 0, 0, .15);
+  --page-header-bg: hsla(0, 100%, 100%, 0.9);
+  --page-header-border: hsl(230, 1%, 86%);
+  --page-header_link-color: hsl(0, 0%, 40%);
+  --page-header_box-shadow-color: rgba(0, 0, 0, .15);
+  --page-header_height: 55px;
+  --theme-switcher-height: 20px;
 
-  // Custom Containers
+  // Alerts
 
   // - generic
-  --customContainerTitle-color: hsl(240, 1%, 28%);
-  --customContainer-border: hsl(230deg, 1%, 66%);
-  --customContainer-bg: hsl(230, 1%, 99%);
-  --customContainer-color: var(--font-color);
+  --custom-containerTitle-color: hsl(240, 1%, 28%);
```

Comment:
This should be `--custom-container-title-color` — but honestly it should be `--alert-title-color`, and all variables here that start with `custom-container` should use `alert` instead.

Reply:
Good call, I never went through and renamed any variables. I'll make sure to take a pass at those as well. Ditching my nonsense naming scheme to go with BEM again here.

Reply:
I'm not saying the variables need to be BEM-like! Just that these should be updated to reflect the new name of the thing. The naming scheme doesn't bother me.

Reply:
Being honest, by the time I read your response, I was already about halfway through renaming them.
So I've just gone ahead and moved everything to BEM style, the consistency is nice though.

And `custom-container` variables have all been moved to `alert` to match class naming